src/main/java/org/openmrs/maven/plugins/model/Server.java

```diff
@@ -0,0 +1,132 @@
+package org.openmrs.maven.plugins.model;
+
+/**
+ * Class for Server model
+ */
+public class Server {
+    private String serverId;
+    private String dbPath;
+    private String dbDriver;
+    private String dbUri;
+    private String dbUser;
+    private String dbPassword;
+    private String interactiveMode;
+    private String version;
+
+    public static class ServerBuilder {
+        private String nestedServerId;
+        private String nestedDbDriver;
+        private String nestedDbUri;
+        private String nestedDbUser;
+        private String nestedDbPassword;
+        private String nestedInteractiveMode;
+        private String nestedVersion;
+
+        public ServerBuilder setNestedInteractiveMode(String nestedInteractiveMode) {
+            this.nestedInteractiveMode = nestedInteractiveMode;
+            return this;
+        }
+
+        public ServerBuilder setNestedVersion(String nestedVersion) {
+            this.nestedVersion = nestedVersion;
+            return this;
+        }
+
+        public ServerBuilder setNestedServerId(String nestedServerId) {
+            this.nestedServerId = nestedServerId;
+            return this;
+        }
+
+        public ServerBuilder setNestedDbDriver(String nestedDbDriver) {
+            this.nestedDbDriver = nestedDbDriver;
+            return this;
+        }
+
+        public ServerBuilder setNestedDbUri(String nestedDbUri) {
+            this.nestedDbUri = nestedDbUri;
+            return this;
+        }
+
+        public ServerBuilder setNestedDbUser(String nestedDbUser) {
+            this.nestedDbUser = nestedDbUser;
+            return this;
+        }
+
+        public ServerBuilder setNestedDbPassword(String nestedDbPassword) {
+            this.nestedDbPassword = nestedDbPassword;
+            return this;
+        }
+
+        public Server build() {
+            return new Server(nestedServerId, nestedVersion, nestedDbDriver, nestedDbUri, nestedDbUser, nestedDbPassword, nestedInteractiveMode);
+        }
+    }
+
+    public Server() {};
+
+    public Server(String serverId, String version, String dbDriver, String dbUri, String dbUser, String dbPassword, String interactiveMode) {
```

Comment:
Using the builder pattern, you usually make constructors protected or private to enforce the use of a builder.